components/tools/FiltersDisplay.js

```diff
@@ -0,0 +1,32 @@
+import { twMerge } from "tailwind-merge";
+
+export default function FiltersDisplay({ checkedDomain = [], setDomain }) {
```

Comment:
```suggestion
export default function FiltersDisplay({ checkedValues = [], setValues }) {
```

Kindly make the name of the variables according to the use/functionality of them in the component.